### DISCUSSION TITLE
Component: Image. Add Option to Disable Toolbar or Individual Action Buttons for preview mode

### DISCUSSION THREAD

**Discussion by jingwei950** [primefaces/community#3276] [PrimeNG]
### Describe the feature you would like to see added
Add options to the PrimeNG `Image` component to disable either the entire action button toolbar or specific individual buttons for image preview mode, such as the zoom, rotate and close buttons. This would allow for greater customization of the component's behavior and appearance.

### Is your feature request related to a problem?
Yes. In some use cases, the action button toolbar provided by the Image component is unnecessary or undesired. For example:

- A minimalist design might not require any toolbar.
- Currently, there is no built-in way to disable these buttons or the toolbar without custom CSS or JavaScript workarounds.
- Action buttons does not work except close button when using it with template.
- Unable to disable toolbar or action buttons using custom CSS when `appendTo="body"` is being used.

### Describe the solution you'd like
Provide input properties to disable the toolbar entirely or disable specific action buttons individually. Examples of the proposed API:

```
<p-image [disableToolbar]="true"></Image />
<p-image [disableZoomIn]="true" [disableCloseButton]="true" />
```

- disableToolbar: Disables the entire toolbar and hides all buttons.
- disableRotateLeft: Disables or hides only the rotate left button.
- disableRotateRight: Disables or hides only the rotate right button.
- disableZoomIn: Disables or hides only the zoom in button.
- disableZoomOut: Disables or hides only the zoom out button.
- disableCloseButton: Disables or hides only the close button.

### Describe alternatives you have considered

1. Custom CSS: Use CSS to hide the toolbar or specific buttons.
Downside: 
- Requires additional maintenance and could break with updates to PrimeNG.

2. Programmatic Workarounds: Manually disable or hide buttons through DOM manipulation.
- Not able to manipulate it from a component/child component when `appendTo=""body"` being used, and not dynamic.
- Inefficient and error-prone.

### Additional context
This feature would be especially beneficial for applications with tailored UI/UX needs, such as image viewers with custom controls or minimalist interfaces. Implementing these properties would increase flexibility and reduce reliance on workaround solutions.

**Comment by aaron-everly** [comment]
+1 on this. Would love to disable the rotation controls without elaborate CSS
